mutt_header.c

```diff
@@ -289,7 +289,12 @@ void mutt_edit_headers(const char *editor, const char *body, struct Email *e,
     bool keep = true;
     size_t plen;
 
-    if (fcc && (plen = mutt_istr_startswith(np->data, "fcc:")))
+    // Check first for "X-Mutt-Fcc" then for "fcc" as it is the more
+    // unambiguous header name.
+    // Note: `plen` is only assigned once due to the short-circuit behaviour of
+    // `||`
```

Comment:
Hmm... You've added this comment in a few places
Are you sure?
If the string in `fcc`, then surely `plen` will be assigned a value twice.
Or can the compiler _really_ optimise that away?

Reply:
My bad that's very poorly written, what I actual meant was this:

>    // Note: If the first string comparison succeeds, `plen` is not overridden
>    // by the second one due to the short-circuit behaviour of `||`.

I added this since for less seasoned C programmers this might be hard to read/understand. (Better safe than sorry)